mission_control/navigator_object_recognition/object_classification.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+from navigator_msgs.msg import PerceptionObject, PerceptionObjects
+from visualization_msgs.msg import Marker
+from geometry_msgs.msg import PointStamped
+from txros import NodeHandle, util
+from nav_msgs.msg import Odometry
+from twisted.internet import defer, reactor
+import numpy as np
+import navigator_tools as nt
+import tf
+
+nh = None
+UP = np.array([0.0, 0.0, 1.0], np.float64)
+EAST, NORTH, WEST, SOUTH = [tf.transformations.quaternion_about_axis(np.pi / 2 * i, UP) for i in xrange(4)]
+
+
+class ObjectClassifier(object):
+
+    def __init__(self):
+        print "init OC"
+        self.MIN_NUM_HITS = 3
+        self.classified_ids = {}
+        self.ids_to_hits = {}
+        self.currently_classifying = False
+        self.all_objects = []
+        self.human_mode = True
+
+    @util.cancellableInlineCallbacks
+    def _init(self):
+        self.nh = yield NodeHandle.from_argv("object_classifier")
+        global nh
+        nh = self.nh
```

Comment:
nh is declared in all the scopes: class, global, and local to the function. For this, you only need class scope and doing it like that is frowned upon.